src/ui/Forms/GetDictionaries.cs

```diff
@@ -77,18 +96,40 @@ private void LoadDictionaryList(string xmlResourceName)
 
                     if (!string.IsNullOrEmpty(downloadLink))
                     {
-                        string name = englishName;
-                        if (!string.IsNullOrEmpty(nativeName))
+                        dictionaryItems.Add(new DictionaryItem
+                        {
+                            Name = englishName,
+                            NativeName = nativeName,
+                            Description = description,
+                            DownloadLink = downloadLink
+                        });
+                    }
+                }
+
+                // ensure ellipses suffix on text overlaps
+                using (var graphics = Graphics.FromHwnd(IntPtr.Zero))
+                {
+                    double comboboxWidth = comboBoxDictionaries.Width;
```

Comment:
Should probably be DropDownWidth, right?

Reply:
> Should probably be DropDownWidth, right?

Changed :) but they return the same value!